docs/eventing/brokers/broker-developer-config-options.md

```diff
@@ -1,6 +1,6 @@
 # Developer configuration options
 
-## Broker configuration example
+## Broker configuration 
```

Comment:
Should we move the full example to the end of the page. So we describe before the configuration options and give then a "full example"?

Reply:
Moving the full example would be better because a section from options were repeated on the Developer Configuration Options. Let me correct